generate/sre-authorization/blacklist.json

```diff
@@ -0,0 +1,32 @@
+[
+    {
+        "apiGroup": "rbac.authorization.k8s.io",
+        "resources": ["clusterrolebindings","rolebindings"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "machineconfiguration.openshift.io",
+        "resources": ["machineconfigs","machineconfigpools","controllerconfigs"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "machine.openshift.io",
+        "resources": ["machines","machinesets"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "config.openshift.io",
+        "resources": ["oauths"],
+        "verbs": ["create","delete","patch","update","deletecollection"]
+    },
+    {
+        "apiGroup": "monitoring.coreos.com",
+        "resources": ["prometheusrules"],
```

Comment:
Shouldn't we add to this black list:
* `prometheus.monitoring.coreos.com`
* `alertmanagers.monitoring.coreos.com`
* `clusteroperators.operatorstatus.openshift.io`

Reply:
We don't manage prometheus or clusteroperators CR via hive so didn't include.  Alertmanagers makes sense to blacklist.  Don't see a problem blacklisting prometheuses too.  Will leave clusteroperators as I don't know if that means we can't upgrade now.